### PR TITLE
Log helper tool output to file, so we can log send

### PR DESCRIPTION
**Pull request by gabriel**
https://keybase.atlassian.net/browse/DESKTOP-1515

### CODE REVIEW

File: osx/Helper/KBLogger.h [hunk 1]
```diff
@@ -0,0 +1,11 @@
+//
+//  KBLogger.h
+//  Keybase
+//
+//  Created by Gabriel on 7/22/16.
+//  Copyright © 2016 Keybase. All rights reserved.
```
Comment (chromakode) [inline osx/Helper/KBLogger.h]: Is this header necessary/correct? Aren't these files covered by the `LICENSE` file at repo root?

File: osx/Helper/KBLogger.m [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+@implementation KBFileLogger
+
+- (id)init {
+  if ((self = [super init])) {
+    // Ensure log file exists
+    NSString * const logPath = @"/Library/Logs/keybase.system.log";
+    if (![[NSFileManager defaultManager] fileExistsAtPath:logPath]) {
+      if (![[NSFileManager defaultManager] createFileAtPath:logPath contents:nil attributes:nil]) {
+        NSLog(@"Unable to log file: %@", logPath);
```
Comment (chromakode) [inline osx/Helper/KBLogger.m]: :sheep: Accidentally a word here? "Unable to create log file"?

**Comment by chromakode** [comment]
:+1:

File: osx/Helper/KBLogger.h (hunk 1 above)
Comment (gabriel) [inline osx/Helper/KBLogger.h]: It's the default produced by Xcode.

**Comment by jzila** [comment]
Please rebase to make CI pass :)

**Pull request merged**